integration/env/common.go

```diff
@@ -24,6 +28,12 @@ const (
 	EnvVarTestedVersion        = "TESTED_VERSION"
 	EnvVarTestDir              = "TEST_DIR"
 	EnvVarVersionBundleVersion = "VERSION_BUNDLE_VERSION"
+
+	// IDChars represents the character set used to generate cluster IDs.
+	// (does not contain 1 and l, to avoid confusion)
```

Comment:
> // (does not contain 1 and l, to avoid confusion)
what about o, and 0 that always make me confuse anyway.

Reply:
AIUI this is aligned with how we generate cluster IDs in the API.